lib/cmds/organization_cmds/export.ts

```diff
@@ -0,0 +1,130 @@
+import Listr from 'listr'
+import { noop } from 'lodash'
+import path from 'path'
+import type { Argv } from 'yargs'
+import { handleAsyncError as handle } from '../../utils/async'
+import { createPlainClient } from '../../utils/contentful-clients'
+import { copyright } from '../../utils/copyright'
+import { cursorPaginate } from '../../utils/cursor-pagninate'
+import { ensureDir, getPath, writeFileP } from '../../utils/fs'
+import { getHeadersFromOption } from '../../utils/headers'
+import { success, log } from '../../utils/log'
+
+module.exports.command = 'export'
+
+module.exports.desc = 'export your organization entities'
+
+module.exports.builder = (yargs: Argv) => {
+  return yargs
+    .usage('Usage: contentful organization export')
+    .option('management-token', {
+      alias: 'mt',
+      describe: 'Contentful management API token',
+      type: 'string'
+    })
+    .option('organization-id', {
+      alias: 'oid',
+      describe: 'ID of Organization with source data',
+      type: 'string',
+      demandOption: true
+    })
+    .option('header', {
+      alias: 'H',
+      type: 'string',
+      describe: 'Pass an additional HTTP Header'
+    })
+    .option('output-file', {
+      alias: 'o',
+      type: 'string',
+      describe:
+        'Output file. It defaults to ./migrations/<timestamp>-<organization-id>.json'
```

Comment:
In the current implementation, this doesn't hold true. We currently print to `stdout` if this flag is not provided.  Maybe we want to the the same as we do for `contentful-export` [where we explicitly define a flag](https://github.com/contentful/contentful-export/blob/main/lib/usageParams.js#L114) for persisting the export in a file (default will print to stdout). 

- If `save-file` is provided, we persist to a file
   - if no `output-file` is not provided, we fall back to a default file name